index.html

```diff
@@ -51,11 +51,11 @@ <h4 class="col1-headers">Backgrounds</h4>
       <div class="bear-images">
         <img class="bear-items" src="assets/bear.png" alt="bear">
         <div class="hats">
-          <img class="top-hat" src="assets/tophat.png" alt="top-hat">
-          <img class="sun-hat" src="assets/sunhat.png" alt="sun-hat">
-          <img class="space-helmet" src="assets/spacehelmet.png" alt="space-helmet">
-          <img class="hair-bow" src="assets/hairbow.png" alt="hair-bow">
-          <img class="crown" src="assets/crown.png" alt="crown">
+          <img class="top-hat hat" src="assets/tophat.png" alt="top-hat">
```

Comment:
Was the hat class added so we can do something similar for the other bear accessories?